v2/cli/login_command.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/bacongobbler/browser"
+	"github.com/brigadecore/brigade/sdk/v2/authx"
+	"github.com/brigadecore/brigade/sdk/v2/restmachinery"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
```

Comment:
Ooooo! I had not seen cli/v2!

Reply:
Awesome thing that v2 adds is ability to make certain flags required. Taking cues from the `az` CLI, I've taken advantage of that quite a bit. If a command takes several arguments, it's _way_ easier to use flags than to try and remember what _position_ each argument belongs in.